src/App/Branding.h

```diff
@@ -26,12 +26,11 @@
 
 #include <string>
 #include <QDomDocument>
-#include <QMap>
-#include <QString>
 #include <QVector>
 
-
+template <class T, class U> class QMap;
```

Comment:
Be careful with forward declarations of template classes. This can lead to serious problems. E.g. for STL classes it's explicitly disallowed to do it because it can lead to undefined behaviour.

In this case it's pointless anyway because Branding.h is only included by two source files.